include/glow/Backends/Backend.h

```diff
@@ -85,6 +86,8 @@ class Backend {
 
   /// Optimize the Function \p F given compilation mode \p mode.
   void optimizeFunction(CompilationMode mode, Function *F);
+  /// \returns true if Backend generated Instruction
```

Comment:
Please name the parameters and mention them here in the doxygen. Also end sentences with a period 😃.